src/coreclr/nativeaot/Runtime/unix/PalCreateDump.h

```diff
@@ -7,5 +7,8 @@
 
 extern bool PalCreateDumpInitialize();
 extern void PalCreateCrashDumpIfEnabled();
+
+#if !defined(HOST_WASM)
 extern void PalCreateCrashDumpIfEnabled(int signal, siginfo_t* siginfo = nullptr, void* exceptionRecord = nullptr);
+#endif // !defined(HOST_WASM)
```

Comment:
I recall CI was seeing failures to the effect of undefined `siginfo_t`; you could forward-declare it to avoid the `#ifdef`.